faq.md

```diff
@@ -5,4 +5,30 @@ css_id: faq
 
 # Frequently asked questions
 
+### **What makes Uptane different from other SOTA security mechanisms?**
 
+
+
+
+
+### **How does Uptane work with other systems and protocols?**
+
+
+
+
+### **Must all signatures be valid for a threshold of signatures to be valid?**
+The threshold requirement of Uptane, mentioned in [Section 5.4.4.3](https://uptane.github.io/uptane-standard/uptane-standard.html#check_root) and in descriptions of other roles, stipulates that a set number of keys are required to sign a metadata file. This is designed to prevent attacks by requiring would-be hackers to compromise multiple keys in order to install malware. However, what happens if this threshold is met, but one or more of the keys is not valid?
+
+As currently configured, the Uptane Standard allows each implementer to decide whether these keys would be accepted. Take the following example:
```

Comment:
I would drop the "As currently configured" bit. I think that would just serve to confuse, because it sounds like we might change it, and I don't think we want to do that.

Reply:
Agree with Patrick's note

```suggestion
The Uptane Standard allows each implementer to decide whether these keys would be accepted. Take the following example:
```